### PR TITLE
[Docs] add some tips for troubleshooting the release

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

**Review by pbench — APPROVED** [review]

**Review by kadhikari — APPROVED** [review]
Great. Thanks a lot @woshilapin

File: readme_release.md [hunk 1]
```diff
@@ -79,5 +82,10 @@ Then the process is less automated (but still, instructions are given):
 	```
 
 # Troubleshooting
-If you run into github's daily limitation, you can easily provide your login/token into the script.
-Search for "rate limit exceeded" in script.
+Note that if the release process fails in the middle, here are a few tips that might help before launching it again:
+
+- check you're on `dev` branch
+- check the branch `release` doesn't exist (might have been created by a previous execution); **or else** remove it
```
Comment (pbougue) [inline readme_release.md]: Are you sure about that one?
To me this is a bug, no?

Comment (woshilapin) [reply readme_release.md]: It might :shrug: Note that I'm only talking about local branches here... maybe I should add a precision about that ?

Comment (woshilapin) [reply readme_release.md]: I removed this line ([011c2ac](https://github.com/hove-io/navitia/pull/3768/commits/011c2ace249921a3ef6ef19e74dd544186bbfbfe)). We might come back to it later to identify if it's a bug or not.

**Review by pbougue — APPROVED** [review]

**Pull request merged**